cvmfs/cvmfs.cc

```diff
@@ -2693,6 +2717,17 @@ static bool RestoreState(const int fd_progress,
         cvmfs::file_system_->RemapCatalogFd(0, new_root_fd);
       }
     }
+
+    if (saved_states[i]->state_id == loader::kStateFuse) {
+      SendMsg2Socket(fd_progress, "Restoring fuse state... ");
+      cvmfs::FuseState *fuse_state =
+        static_cast<cvmfs::FuseState *>(saved_states[i]->state);
+      if (!fuse_state->cache_symlinks)
+        cvmfs::mount_point_->DisableCacheSymlinks();
```

Comment:
i think we need also an `EnableCacheSymlink` function, as the default is `false`. i cannot see here how we can set it to `true`